conda_forge_tick/settings.py

```diff
@@ -0,0 +1,106 @@
+from typing import Annotated
+
+from pydantic import Field
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+ENVIRONMENT_PREFIX = "CF_TICK_"
+"""
+All environment are expected to be prefixed with this.
+"""
+
+ENV_CONDA_FORGE_ORG = ENVIRONMENT_PREFIX + "CONDA_FORGE_ORG"
```

Comment:
not really pretty but i also can't think of another way. especially with the test this is fine imo

Reply:
yeah I stopped spending time on how to make this better :D